src/app/todo/interfaces/todo.ts

```diff
@@ -1,6 +1,6 @@
 export interface Todo {
   id: string;
-  name: string;
+  taskName: string;
```

Comment:
don't change interface, it's a contract between you and the API
to change it, API must change as well